packages/expo-facebook/android/src/main/java/expo/modules/facebook/FacebookModule.java

```diff
@@ -85,18 +130,29 @@ public void onInitialized() {
         }
       });
     } catch (Exception e) {
-      promise.reject(e);
+      promise.reject("ERR_FACEBOOK_UNINITIALIZED", "An error occurred while trying to initialize a FBSDK app", e);
```

Comment:
I'm having second thoughts on whether `ERR_FACEBOOK_UNINITIALIZED` works well here — in other places it means "don't call this method, you haven't initialized the SDK yet" and here we mean something more along the lines of `ERR_FACEBOOK_MISCONFIGURED` "the SDK could not have been initialized properly, check the configuration".